js/forms.js

```diff
@@ -637,8 +637,16 @@
         }
       });
 
-      $newSelect.on('blur', function() {
-        if (!multiple) {
+      $newSelect.on('blur', function(e) {
+        var that = this;
+
+        $(this).find(' ~ .dropdown-content span').off('click');
+        $(this).find(' ~ .dropdown-content span').on('click', function() {
+          $(that).trigger('close');
+        });
+
+        var containers = $('.select-dropdown');
```

Comment:
I would try to filter down the selector to match only this .select-dropdown and not all of those present in the page. But it's minor as currently it's not possible to have more than one dropdown visible at a time.
Please wait for @WiktorStarczewski review to merge

Reply:
Dont use global selectors, ever, please.